src/Tools/dotnet-dev-jwts/src/Commands/ClearCommand.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal sealed class ClearCommand
+{
+    public static void Register(CommandLineApplication app)
+    {
+        app.Command("clear", cmd =>
+        {
+            cmd.Description = "Delete all issued JWTs for a project";
+
+            var projectOption = cmd.Option(
+                "--project",
+                "The path of the project to operate on. Defaults to the project in the current directory.",
+                CommandOptionType.SingleValue);
+
+            var forceOption = cmd.Option(
+                "--force",
+                "Don't prompt for confirmation before deleting JWTs",
+                CommandOptionType.NoValue);
+
+            cmd.HelpOption("-h|--help");
+
+            cmd.OnExecute(() =>
+            {
+                return Execute(projectOption.Value(), forceOption.HasValue());
+            });
+        });
+    }
+
+    private static int Execute(string projectPath, bool force)
+    {
+        var project = DevJwtCliHelpers.GetProject(projectPath);
+        if (project == null)
+        {
+            Console.WriteLine($"No project found at {projectPath} or current directory.");
```

Comment:
This message should probably be updated. This will be null if there are multiple projects in the path too, for instance. And there's the whole question of whether the path was implicit or explicitly specified.

Reply:
(same applies to everywhere else this pattern is used)